index.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const greet = require('./lib/greet.js');
+
+greet.sayHi('Gary');
+greet.sayBye();
+greet.greetStranger();
```

Comment:
Great work writing a separate method to be able to handle the user input for the hello module!